modules/microshift-embed-images-offline-use.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// microshift_running_applications/embed-microshift-apps-on-rhel-edge.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-embed-images-offline-use_{context}"]
+= Embedding workload container images for offline use
+
+You can embed container images to use in devices at the edge that do not have any network connection. You must create a new container, mount the ISO and then copy the contents into the FileSystem.
```

Comment:
I would minimize this section contents by:
1. Showing an example how user image references can be extracted from manifests.
1. Referring to an existing documentation section about embedding container images into ISO

Reply:
as said above, its really not that easy to figure this out if your are not a certified RHEL admin, so I would prefer to keep it here, to make it easier for users to consume and follow.

Reply:
See my above comments as well :)